protobuf-java-converter/src/main/kotlin/io/bkbn/kompendium/protobufjavaconverter/converters/FieldDescriptiorConverters.kt

```diff
@@ -134,7 +136,8 @@ fun handleMapField(
  */
 fun fromTypeToSchema(
   javaProtoField: Descriptors.FieldDescriptor,
-  cache: MutableMap<String, JsonSchema> = mutableMapOf()
+  cache: MutableMap<String, JsonSchema> = mutableMapOf(),
+  componentSlug: String = "#/components/schemas"
```

Comment:
@unredundant is there a way we can make this public? I don't have access to the type any more when I need to create the ref ( ye could hack around with reflection but that's a bit :/ )

Reply:
Make what public? This custom component schema slug?

Reply:
ye the slug string so that I can reuse it here instead of having to redefine `"#/components/schemas"`

Reply:
sure yea, I have no objections if you want to just drop the `private` from the COMPONENT_SLUG variable and just use that

Reply:
Once you do that, if you could add a couple full tests (generate the whole schema) that would be awesome, just to help prevent regressions.  If you look at the `KompendiumTest` file you should find some useful helpers that you should be able to leverage.

Reply:
finally had some time to fix it see if you agree and if missed something